### DISCUSSION TITLE
No R/S/T Voltage?

### DISCUSSION THREAD

**Discussion by ICEAndreas** [nathanmarlor/foxess_modbus#646] [Support] [answered]
Hi,
is there no R/S/T Voltage available? I have only current.

**Comment by FozzieUK1** [comment, marked as answer]
Yes you should have them on a 3 phase install.

Check in Settings, Devices & Services in the Foxess Modbus integration, there are a number of entities left disabled as default which will show at the bottom of the list as 'xx entities not shown' - click on that, find the ones you want and enable them

**Reply by ICEAndreas** [reply]
Thanks.
